packages/Chem/src/chem-searches.ts

```diff
@@ -6,14 +6,15 @@ import {
   defaultMorganFpLength,
   defaultMorganFpRadius,
   Fingerprint,
-  rdKitFingerprintToBitArray
+  rdKitFingerprintToBitArray,
 } from './utils/chem-common';
 import BitArray from '@datagrok-libraries/utils/src/bit-array';
 import {tanimotoSimilarity} from '@datagrok-libraries/utils/src/similarity-metrics';
 import {assure} from '@datagrok-libraries/utils/src/test';
 
 function _chemFindSimilar(molStringsColumn: DG.Column,
-  queryMolString: string, settings: { [name: string]: any }) {
+  queryMolString: string, settings: { [name: string]: any })
+  :DG.DataFrame | 0 {
```

Comment:
I don't think it returns 0
Also, the return type fits on the original line (max width is at least 120 characters)